code/game/gamemodes/dynamic/dynamic_rulesets_latejoin.dm

```diff
@@ -216,3 +216,41 @@
 	new_heretic.knowledge_points = min(new_heretic.knowledge_points, 5)
 
 	return TRUE
+
+//////////////////////////////////////////////
+//                                          //
+//           LATEJOIN CHANGELING            //
+//                                          //
+//////////////////////////////////////////////
+
+/datum/dynamic_ruleset/latejoin/stowaway_changeling
+	name = "Stowaway Changeling"
+	antag_datum = /datum/antagonist/changeling
+	antag_flag = ROLE_STOWAWAY_CHANGELING
+	antag_flag_override = ROLE_CHANGELING
+	protected_roles = list(
+		JOB_CAPTAIN,
+		JOB_DETECTIVE,
+		JOB_HEAD_OF_PERSONNEL,
+		JOB_HEAD_OF_SECURITY,
+		JOB_PRISONER,
+		JOB_SECURITY_OFFICER,
+		JOB_WARDEN,
+	)
+	restricted_roles = list(
+		JOB_AI,
+		JOB_CYBORG,
+	)
+	required_candidates = 1
+	weight = 2
+	cost = 12
+	requirements = list(101,101,40,40,20,20,10,10,10,10)
+	repeatable = TRUE
+
+/datum/dynamic_ruleset/latejoin/stowaway_changeling/execute()
+	var/mob/picked_mob = pick(candidates)
+	assigned += picked_mob.mind
+	picked_mob.mind.special_role = antag_flag
+	var/datum/antagonist/heretic/new_changeling = picked_mob.mind.add_antag_datum(antag_datum)
```

Comment:
`heretic`?

Also why do you need to save this as a variable?

Reply:
No reason to save it as a variable you're correct. Just some lame copy+paste code I didn't look at too carefully.